src/main/resources/attackvectors/BlindSQLInjectionVulnerabilityPayload.properties

```diff
@@ -0,0 +1,29 @@
+BLIND_SQL_INJECTION_PAYLOAD_LEVEL_1=As blind SQL injection doesn't show any internal details like errors or table data hence it allows a hacker to check if the web application is vulnerable to SQL injection by sending <b>true</b> and <b>false</b> statements.<br/>\
+ <b>How to exploit this level?</b><br/>\
+  Insert those in textbox<br/>\
+  <u>For true statement</u><br/>\
+  <code>100 OR 2=2</code><br/>\
+  Expected result<br/>\
+  <i>{ "isCarPresent": true }</i><br/>\
+We know <i>100 id</i> is not present in the table but still, it is returning <i>true</i>. it means that the input is evaluated to <i>true</i> and the backend has returned that <i>car</i> is present, which could indicate vulnerability.<br/>\
+  <u>For false statement</u><br/>\
+  <code>1 AND 1=2</code><br/>\
+  Expected result<br/>\
+  <i>{ "isCarPresent": false }</i> <br/>\
+  We know that car with <i>id = 1</i> is present but because of second part backend returns <i>false</i>.
+BLIND_SQL_INJECTION_PAYLOAD_LEVEL_2=As blind SQL injection doesn't show any internal details like errors or table data hence it allows a hacker to check if the web application is vulnerable to SQL injection by sending <b>true</b> and <b>false</b> statements.<br/>\
+ <b>How to exploit this level?</b><br/>\
+  Following input <i>1 OR 2=2</i> and <i>1 AND 1=2</i> returns <i>System Error Occurred. please check logs</i> but doesn't give any details like table name or queries.<br/>\
+  Additional security was added - value of <i>id</i> parameter is surrounded by quote in that way <code>id='value'</code> <br/>\
+  So now we have to insert those in textbox<br/>\
+  <u>For true statement</u><br/>\
+  <code>100'OR '1'='1</code><br/>\
+  Expected result<br/>\
+  <i>{ "isCarPresent": true }</i><br/>\
+  Like in previous example, server will return <i>true</i>. Because of clever use quote sign, we was able to outsmart security.<br/>\
+  <u>For false statement</u><br/>\
+  <code>1'AND '1'='2</code><br/>\
+  Expected result<br/>\
+  <i>{ "isCarPresent": false }</i><br/>\
+  <i>False</i> is returned despite the fact that <i>id=1</i> is present in the table.
```

Comment:
Similarly hackers can use the same technique to retrieve entire data from database.